src/components/views/elements/AddressSelector.js

```diff
@@ -16,16 +16,19 @@ limitations under the License.
 
 'use strict';
 
-var React = require("react");
-var sdk = require("../../../index");
-var classNames = require('classnames');
+const React = require("react");
+const sdk = require("../../../index");
+const classNames = require('classnames');
+const InviteAddressType = require("./AddressTile");
```

Comment:
this looks dodge - the default export of AddressTile is the react component, not InviteAddressType.